website/routes.py

```diff
@@ -134,6 +134,7 @@ def get_globals():
         'osf_url': settings.INTERNAL_DOMAIN,
         'waterbutler_url': settings.WATERBUTLER_URL,
         'login_url': cas.get_login_url(request_login_url),
+        'sign_up_url': util.web_url_for('auth_register', _absolute=True) + '?next=' + request_login_url,
```

Comment:
As per COSDev, prefer str.format to concatenation.

```suggestion
        'sign_up_url': util.web_url_for('auth_register', _absolute=True) + '?next={}'.format(request_login_url),
```

I can make this change before I merge.

Reply:
Sorry I keep doing this ugh!